### ISSUE TITLE
Basic progress spinner of 2.0.0 Beta 12 is always a circle in Internet Explorer and Edge

### ISSUE THREAD

**Issue by zijianhuang** [angular/material2#7580]
#### Bug:

`<mat-spinner>` or `<mat-progress-spinner mode="indeterminate">` is giving a circle rather than a spinning one in Internet Explorer and Microsoft Edge

#### What is the expected behavior?

Spinning.

#### What is the current behavior?

Static circle

#### What are the steps to reproduce?

Just open https://material.angular.io/components/progress-spinner/overview in IE or Edge.

The spinner is working fine in Chrome, Firefox and Safari 

In Beta 11 and older, the spinner is working fine in IE and Edge.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular 4.4.4, NG **Material 2.0.0-beta.12**, TypeScript 2.5.3, IE 11.608.15063, Edge 40.15063

**Comment by shalugin** [comment]
I can't even change tab from "Overview" to "API".
```

vendor.c40d3842034d42a06e10.bundle.js:252 ERROR TypeError: Invalid event target
    at Function.e.setupSubscription (vendor.c40d3842034d42a06e10.bundle.js:15)
    at e._subscribe (vendor.c40d3842034d42a06e10.bundle.js:15)
    at e.t._trySubscribe (vendor.c40d3842034d42a06e10.bundle.js:1202)
    at e.t.subscribe (vendor.c40d3842034d42a06e10.bundle.js:1202)
    at t.call (vendor.c40d3842034d42a06e10.bundle.js:1146)
    at t.subscribe (vendor.c40d3842034d42a06e10.bundle.js:1202)
    at l.ngOnInit (main.7ead62fe291d539d45c5.bundle.js:1)
    at hn (vendor.c40d3842034d42a06e10.bundle.js:336)
    at rr (vendor.c40d3842034d42a06e10.bundle.js:371)
    at er (vendor.c40d3842034d42a06e10.bundle.js:371)
```

And I see endless queries in developer console.

**Comment by zijianhuang** [comment]
@shalugin, apparently https://material.angular.io/ has just become a SPA, however, if you access to a page like https://material.angular.io/components/button/overview directly, you lost the side bar, and the top navigation bar is not working well with errors similar to what you had shown. Apparently the server side rewrite of the SPA is having some problems.

Would you raise a ticket?

**Comment by shalugin** [comment]
@zijianhuang I made a new ticket #7583

**Comment by crisbeto** [comment]
Edge is supported, however we have a fallback animation. That being said the fallback animation doesn't work at the moment and will be fixed by https://github.com/angular/material2/pull/7599.

**Comment by josephperrott** [comment]
Closing as fixed by #7599
